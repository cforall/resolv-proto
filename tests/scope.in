T f T
%%
f( #int )

{
	#int i
	%%
	f( i() )
}

{
	#float i
	%%
	f( i() )
}

{
	#int f
	%%
	f( f() )
}

{
	#float f
	%%
	f( f() )
}