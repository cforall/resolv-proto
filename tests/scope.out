
[#int / (0,1,1,0,0)] f( #int )

[#int / (0,1,1,0,0)] f( i( ) )

[#float / (0,1,1,0,0)] f( i( ) )

[#int / (0,1,1,0,0)] f( f( ) )

[#float / (0,1,1,0,0)] f( f( ) )
